solidity/tests/WebAuthn_forge/script/DeployElliptic.s.sol

```diff
@@ -86,14 +86,14 @@ contract FCL_all_wrapper {
 contract MyScript is BaseScript {
     function run() external broadcast returns (address addressOfLibrary) {
         // deploy the library contract and return the address
-        addressOfLibrary = address(new FCL_ecdsa_wrapper{salt:0}());
+        addressOfLibrary = address(new FCL_ecdsa_wrapper{salt: 0}());
     }
 }
 
 contract Script_Deploy_FCL_all is BaseScript {
     function run() external broadcast returns (address addressOfLibrary) {
         // deploy the library contract and return the address
-        addressOfLibrary = address(new FCL_all_wrapper{salt:0}());
+        addressOfLibrary = address(new FCL_all_wrapper{salt: 0}());
```

Comment:
Just for the record, those changes were required to pass the formatting verification on the CI.

Reply:
https://github.com/rdubois-crypto/FreshCryptoLib/pull/49/commits/74140397833c8cb318c698354721c88789cf069f#diff-80c8f8afb06afd4ea8342b2f9033a56dc70b73f6350e373173efbd3499c212bb